clang/test/3C/base_subdir/unwritable_rootcauses.c

```diff
@@ -0,0 +1,16 @@
+// RUN: cd %S
+// RUN: 3c -alltypes -addcr -output-dir=%t.checked/base_subdir -warn-all-root-cause %s -- -Xclang -verify=unwritable-expected
+
+#include "../unwritable_rootcauses.h"
+#include "../root_cause.c"
+
+int foo(void) { 
+  int *a = ptr;
+  int *z = a;
+  int* q = f1();
```

Comment:
```suggestion
  int *q = f1();
```
Please make the formatting consistent.  You can try using `clang-format` if that's less work and doesn't mess up too many other things.